contributors/design-proposals/storage/volume-topology-scheduling.md

```diff
@@ -862,6 +862,13 @@ processing later in the priority and bind functions.
    temporarily cache this decision in the PVC per Node.
 8. Otherwise return false.
 
+Update NoVolumeZoneConflict predicate function to skip unbound PVC if StorageClass binding
```

Comment:
This may fit better under the "Other predicates" section

Reply:
Updated.